### ISSUE TITLE
ModuleNotFoundError: No module named 'tensorflow.contrib'. Latest version of TensorFlow not working with TFLearn?

### ISSUE THREAD

**Issue by ArshaHaryoko** [tensorflow/tensorflow#36419]
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Windows 10
- TensorFlow installed from (source or binary): binary
- TensorFlow version: 2.0.0
- Python version: 3.7, 3.6.10 on virtual environment
- Installed using virtualenv? pip? conda?: conda
- CUDA/cuDNN version: CUDA 9.0 / cuDNN 7.6.5
- GPU model and memory: NVIDIA GeForce GTX 1660 SUPER, 16GB

So iv'e been trying to install Tensorflow for a while and conda seems like the method that worked best for me, iv'e pretty much got it working in good order, i tried messing around a bit then decided to follow a tutorial which prompted to install tflearn

tflearn is what seems to be causing the problem, i saw an issue over at their repository. But i'am not certain of the steps i should be taking to resolve the issue, heres the traceback;
```
 File "main.py", line 6, in <module>
   import tflearn
 File "C:\Users\User\Anaconda3\envs\chatbot\lib\site-packages\tflearn\__init__.py", line 4, in <module>
   from . import config
 File "C:\Users\User\Anaconda3\envs\chatbot\lib\site-packages\tflearn\config.py", line 5, in <module>
   from .variables import variable
  File "C:\Users\User\Anaconda3\envs\chatbot\lib\site-packages\tflearn\variables.py", line 7, in <module>
   from tensorflow.contrib.framework.python.ops import add_arg_scope as contrib_add_arg_scope
 
ModuleNotFoundError: No module named 'tensorflow.contrib'
```
I'am not certain if i should downgrade tensorflow, but to what version? Or are there steps i need to take to fix the issue?

Thanks in advance,

**Comment by mihaimaruseac** [comment]
`tf.contrib` has been removed from TF2.0 onwards, especially since it was not properly maintained

We recommend switching to alternatives or downgrading to 1.15

**Comment by synergy178** [comment]
> `tf.contrib` has been removed from TF2.0 onwards, especially since it was not properly maintained
> 
> We recommend switching to alternatives or downgrading to 1.15

@mihaimaruseac  TF 1.15 seems not to have the tf.contrib module anymore too. It's still available in TF<=1.14, but some models are trained not properly with that version. For example the inputs layer shows following (?,?,?,3). While it should be (None,None,None,3) for image data. It's very frustrating currently because at least for me everything worked perfectly with 1.15 until Tuesday, March,17. 
Seems like something broke or someone removed the tf.contrib module in 1.15.

**Comment by mihaimaruseac** [comment]
1.15 should have it. What error do you get? What is the output of `pip list`?
